packages/lesswrong/components/posts/EALargePostsItem.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useRef } from "react";
```

Comment:
This is copied from the EA best of page and then modified in a few places. The version on the best of page will be deleted as part of the redesign there that's currently in code review.